app/models/claim.rb

```diff
@@ -304,6 +302,10 @@ def decision_undoable?
 
   private
 
+  def ecp_skip_validation
```

Comment:
Methods that return boolean should probably end in `?`. Better name would be something like `is_ecp_policy?`.

Reply:
Fixed as suggested. Its temporary anyway until such time as we have a valid claim. Then this can be removed